android/src/main/java/com/th3rdwave/safeareacontext/SafeAreaView.java

```diff
@@ -45,12 +45,12 @@ private EdgeInsets getSafeAreaInsets() {
       int statusBarHeight = 0;
       int resourceId = getResources().getIdentifier("status_bar_height", "dimen", "android");
       if (resourceId > 0) {
-        statusBarHeight = getResources().getDimensionPixelSize(resourceId);
+        statusBarHeight = (int) (getResources().getDimension(resourceId) / getResources().getDisplayMetrics().density);
```

Comment:
Can you use the helper from react native `PixelUtil.toDIPFromPixel`

Reply:
Successfully changed to `PixelUtil.toDIPFromPixel`